ledger/participant-integration-api/src/main/scala/platform/store/dao/events/package.scala

```diff
@@ -68,7 +68,7 @@ package object events {
       .splitWhen(_._2)
       .map(_._1)
       .fold(Vector.empty[A])(_ :+ _)
-      .mergeSubstreams
+      .concatSubstreams
```

Comment:
Could I ask you to put a comment here that `concatSubstreams` is crucial for delivery of the groups in the correct order?

Reply:
Sure, thanks for raising this.

Reply:
Addressed by https://github.com/digital-asset/daml/pull/8336/commits/9af9baff0d7032327028da6ebfd9c1515eb237c9